resources/sushy-tools/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.7
+
+RUN apt update && apt install -y libvirt-dev && cd /root/ && \
+    git clone https://opendev.org/openstack/sushy-tools.git && \
```

Comment:
you could also do
pip install git+https://opendev.org/openstack/sushy-tools.git

Reply:
done, thank you